src/electron/index.ts

```diff
@@ -169,7 +169,16 @@ function setupWindow(): void {
   // user clicked on one of the Privacy, Terms, etc., links. These should
   // open in the user's browser.
   mainWindow.webContents.on('will-navigate', (event: Event, url: string) => {
-    shell.openExternal(url);
+    try {
+      const parsed: URL = new URL(url);
+      if (parsed.protocol === 'http:' || parsed.protocol === 'https:' || parsed.protocol === 'macappstore:') {
```

Comment:
Where do we link to the mac store? Should we be linking to the https url instead?

Reply:
This is basically just taken from the manager, for speed. But I guess we don't host the manager in the mac store? So I'll remove it, sure.